packages/cli/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "grapesjs-cli",
+  "version": "4.1.3",
+  "description": "GrapesJS CLI tool for the plugin development",
+  "bin": {
+    "grapesjs-cli": "dist/cli.js"
+  },
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "BUILD_MODE=production webpack --config ./webpack.cli.ts",
+    "build:watch": "webpack --config ./webpack.cli.ts --watch",
+    "lint": "eslint src",
+    "patch": "npm version patch -m 'Bump v%s'",
+    "test": "jest"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GrapesJS/grapesjs.git"
+  },
+  "keywords": [
+    "grapesjs",
+    "plugin",
+    "dev",
+    "cli"
+  ],
+  "author": "Artur Arseniev",
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "@babel/core": "7.25.2",
+    "@babel/plugin-transform-runtime": "7.25.4",
+    "@babel/preset-env": "7.25.4",
+    "@babel/runtime": "7.25.6",
+    "babel-loader": "9.1.3",
+    "chalk": "^4.1.2",
+    "core-js": "3.38.1",
+    "dts-bundle-generator": "^8.0.1",
+    "html-webpack-plugin": "5.6.0",
+    "inquirer": "^8.2.5",
+    "listr": "^0.14.3",
+    "lodash.template": "^4.5.0",
+    "rimraf": "^4.1.2",
+    "spdx-license-list": "^6.6.0",
+    "terser-webpack-plugin": "^5.3.10",
+    "webpack": "5.94.0",
+    "webpack-cli": "5.1.4",
+    "webpack-dev-server": "5.1.0",
+    "yargs": "^17.6.2"
+  },
+  "devDependencies": {
+    "@types/webpack-node-externals": "^3.0.0",
+    "copy-webpack-plugin": "^11.0.0",
+    "fork-ts-checker-webpack-plugin": "^8.0.0",
+    "webpack-node-externals": "^3.0.0"
+  }
```

Comment:
A lot of these packages are fairly out of date. I think we should keep them as they are, and before upgrading them add more comprehensive tests to the CLI so we update each with confidence.